tensorflow/python/ops/histogram_ops.py

```diff
@@ -62,17 +62,14 @@ def histogram_fixed_width_bins(values,
 
   Examples:
 
-  ```python
-  # Bins will be:  (-inf, 1), [1, 2), [2, 3), [3, 4), [4, inf)
-  nbins = 5
-  value_range = [0.0, 5.0]
-  new_values = [-1.0, 0.0, 1.5, 2.0, 5.0, 15]
-
-  with tf.compat.v1.get_default_session() as sess:
-    indices = tf.histogram_fixed_width_bins(new_values, value_range, nbins=5)
-    variables.global_variables_initializer().run()
-    sess.run(indices) # [0, 0, 1, 2, 4, 4]
-  ```
+  >>> # Bins will be:  (-inf, 1), [1, 2), [2, 3), [3, 4), [4, inf)
+  ...
+  >>> nbins = 5
+  >>> value_range = [0.0, 5.0]
+  >>> new_values = [-1.0, 0.0, 1.5, 2.0, 5.0, 15]
+  >>> indices = tf.histogram_fixed_width_bins(new_values, value_range, nbins=5)
+  >>> print(indices)
```

Comment:
Apparently our internal linter is stupid and doesn't like print in comments (it warns about debug prints). Thankfully you can just replace this with `indices` and it'll work just the same.

Can you fix here and in the other print statements?